src/main/java/club/minnced/discord/webhook/send/WebhookMessageBuilder.java

```diff
@@ -424,45 +421,38 @@ public WebhookMessage build() {
      * @param  message
      *         The message
      *
-     * @throws NullPointerException
-     *         If null is provided
-     *
      * @return WebhookMessageBuilder with the converted data
      */
     @NotNull
     public static WebhookMessageBuilder fromJDA(@NotNull net.dv8tion.jda.api.entities.Message message) {
+        return fromJDA(new MessageCreateBuilder()
+                .applyMessage(message)
+                .setFiles(message.getAttachments().stream()
+                        .map(attachment -> FileUpload.fromData(attachment.getProxy().download().join(), attachment.getFileName()))
+                        .collect(Collectors.toList()))
```

Comment:
I'm against implicitly downloading files from messages here. Please remove all of that functionality.

Reply:
Removed